code/datums/outfits/outfit_admin.dm

```diff
@@ -71,7 +78,7 @@
 /datum/outfit/admin/syndicate_infiltrator/equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	. = H.equip_syndicate_infiltrator(0, 20, FALSE)
 	H.sec_hud_set_ID()
-
+	H.faction += "syndicate"
```

Comment:
```suggestion
	H.faction |= "syndicate"
```

Reply:
This can be comitted into your branch with no changes tourte